packages/prover/test/e2e/web3_provider.test.ts

```diff
@@ -54,7 +54,7 @@ describe("web3_provider", function () {
           mutateProvider: false,
         });
 
-        const web3 = new Web3(nonVerifiedProvider);
+        const web3 = new Web3(verifiedProvider); //TODO: type is not compatible
```

Comment:
```suggestion
        const web3 = new Web3(nonVerifiedProvider); 
```

Reply:
Because we want to get the accounts via normal RPC and then verify it through our `ELRpc` object.

Reply:
the examples in the README show otherwise

Reply:
a per README we pass the `verifiedProvider` to `Web3` but the types are not compatible, the example in the README does not work

https://github.com/ChainSafe/lodestar/blob/14855ea72fa7c46c984ed13078aac1c21858b993/packages/prover/README.md?plain=1#L54